### PR TITLE
Precreate job clusters in master docker image

### PR DESCRIPTION
**Pull request by neerajrj**
### Context

Prepoluate the master docker image with job clusters

### Checklist

- [ x] `./gradlew build` compiles code correctly
- [x ] Added new tests where applicable
- [ x] `./gradlew test` passes all tests
- [ x] Extended README or added javadocs where applicable
- [ x] Added copyright headers for new files from `CONTRIBUTING.md`

### CODE REVIEW

File: buildDockerImage.sh [hunk 1]
```diff
@@ -4,6 +4,6 @@
 ./gradlew clean installDist
 
 # build the Docker image that packages mantis-control-plane
-docker build -t netflixoss/mantiscontrolplaneserver .
+docker build -t dev/mantiscontrolplaneserver .
```
Comment (jeffchao) [inline buildDockerImage.sh:7]: why `dev`?

File: Dockerfile [hunk 2]
```diff
@@ -22,6 +22,14 @@ RUN mkdir -p /apps/mantis/mantis-control-plane-server/logs
...
 RUN mkdir -p /tmp/MantisArchive
 
+COPY docker/SharedMrePublishEventSource /tmp/MantisSpool/namedJobs
+COPY docker/SineFunction /tmp/MantisSpool/namedJobs
+COPY docker/SyntheticSourceJob /tmp/MantisSpool/namedJobs
+COPY docker/TwitterSample /tmp/MantisSpool/namedJobs
+COPY docker/GroupBySample /tmp/MantisSpool/namedJobs
+COPY docker/JobConnectorSample /tmp/MantisSpool/namedJobs
+COPY docker/KafkaSourceJob /tmp/MantisSpool/namedJobs
+
```
Comment (jeffchao) [inline Dockerfile:32]: Should these files specify a `.json` file extension?

**Review by jeffchao — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (neerajrj) [reply Dockerfile:32]: These are the repository artifacts they don't have a .json suffix

File: buildDockerImage.sh (hunk 1 above)
Comment (neerajrj) [reply buildDockerImage.sh:7]: So as to not confuse with the images on docker-hub when developing locally.

Comment (jeffchao) [reply buildDockerImage.sh:7]: Ok we could probably add a release script to docker hub in the future. 👍

**Pull request merged**